roles/webserver/templates/nginx_unicorn.j2

```diff
@@ -18,7 +18,7 @@ server {
 
   include {{ app }}_ssl;
 
-  server_name www.{{ domain }};
+  server_name {{sub_domain}} {{ domain }};
```

Comment:
Yes. Domain is fine, but `hostname` is better. As you said, it's the actual term for that part of the url. It's a little more obtuse in terms of the general population, but the `www.` preceding it makes it quite clear what is being referred to.